Dockerfile

```diff
@@ -1,4 +1,12 @@
-FROM concourse/buildroot:curl
+FROM progrium/busybox
```

Comment:
We shouldn't change this - all core resources should be based on `concourse/buildroot:some-tag`. Also this particular image is deprecated and I've seen parts of it break (packages list is out of date or something).